content/Watsonx/WatsonxData/107.md

```diff
@@ -3,6 +3,7 @@ title: '107: Offloading Data from a Data Warehouse'
 timeToComplete: 45
 updated: 2023-09-20
 ---
+<QuizAlert text='Heads Up! Quiz material will be flagged like this!' />
```

Comment:
Unnecessary if there are no quiz alerts on this page?

Reply:
removed